subprojects/robotpy-wpilib/wpilib/timedrobotpy.py

```diff
@@ -0,0 +1,396 @@
+from sys import argv
+from typing import Any, Callable, Iterable, ClassVar
+from heapq import heappush, heappop, _siftup
+from hal import (
+    report,
+    initializeNotifier,
+    setNotifierName,
+    observeUserProgramStarting,
+    updateNotifierAlarm,
+    waitForNotifierAlarm,
+    stopNotifier,
+    tResourceType,
+    tInstances,
+)
+from wpilib import RobotController
+import wpimath.units
+
+from .iterativerobotpy import IterativeRobotPy
+
+_getFPGATime = RobotController.getFPGATime
```

Comment:
I think putting this in the robot instance, to avoid the global lookup, will probably be faster?